api/PaymentRequest.json

```
@@ -604,36 +605,35 @@
           }
         }
       },
-      "requestId": {
+      "shippingAddress": {
         "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/PaymentRequest/requestId",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/PaymentRequest/shippingAddress",
           "support": {
             "chrome": {
-              "version_added": false
+              "version_added": "61"
             },
             "chrome_android": {
-              "version_added": false
+              "version_added": "53"
             },
             "edge": {
-              "version_added": "≤18",
-              "version_removed": "79"
+              "version_added": "15"
             },
             "firefox": {
               "version_added": false,
-              "notes": "Available only in nightly builds."
+              "notes": "Available only in nightly builds as <code>shippingAddress</code>."
```

This note is now invalid:

```suggestion
              "flags": [
                {
                  "type": "preference",
                  "name": "dom.payments.request.enabled"
                }
              ]
```